src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/add.jsp").forward(req, resp);
```

Comment:
in checklist mentioned not to use plural forms for package names. Rename ALL your package names to a single form
```suggestion
        req.getRequestDispatcher("/WEB-INF/view/car/add.jsp").forward(req, resp);
```

Reply:
This is not package.
this is directory
Example from checklist.

Good URLs naming in your project:

/cars/drivers/add - (add driver to car)
/cars/drivers/delete - (remove driver from car)
/cars/drivers/ - (get all drivers from car)
/drivers/add
/drivers/delete
/manufacturers/delete